cognite/client/data_classes/annotation_types/primitives.py

```diff
@@ -21,15 +21,17 @@ def dump(self, camel_case: bool = False) -> Dict[str, Any]:
         Returns:
             Dict[str, Any]: A dictionary representation of the instance.
         """
-        return {
-            to_camel_case(key)
-            if camel_case
-            else key: value.dump(camel_case=camel_case)
-            if hasattr(value, "dump")
-            else value
-            for key, value in self.__dict__.items()
-            if value not in EXCLUDE_VALUE and not key.startswith("_")
-        }
+        dumped = {}
+        for k, v in vars(self).items():
+            if v is None or k.startswith("_"):
```

Comment:
Only potential risk here is if EXCLUDE VALUE changes, we won't actually exclude it here.

Although I view the chance of changing that to be low - up to you to consider adding a test for it.

Reply:
I'm removing `EXCLUDE_VALUE` in another PR, which this PR is split out from!